hello.c

```diff
@@ -2,6 +2,9 @@
 
 int main()
 {
+    //notion: in the main function,it is necessary to describe the client's function however simple the code is.
```

Comment:
I see what you mean, but I don't think it's necessary to keep it.

Reply:
It is my error to write this sentence in a wrong place,it is more reasonable to  place it in the comment.